crawlers/server/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+)
+
+var MOONSTREAM_IPC_PATH = os.Getenv("MOONSTREAM_DB_URI")
+
+type GethEthSyncingResponse struct {
+	CurrentBlock string `json:"currentBlock"`
+}
+
+type GethResponse struct {
+	Result GethEthSyncingResponse `json:"result"`
+}
+
+type PingGethResponse struct {
+	Status       string `json:"status"`
+	CurrentBlock string `json:"current_block"`
+}
+
+type PingResponse struct {
+	Status string `json:"status"`
+}
+
+func ping(w http.ResponseWriter, req *http.Request) {
+	log.Printf("%s, %s, %q", req.RemoteAddr, req.Method, req.URL.String())
+
+	w.Header().Set("Content-Type", "application/json")
+	response := PingResponse{Status: "ok"}
+	json.NewEncoder(w).Encode(response)
+}
+
+func pingGeth(w http.ResponseWriter, req *http.Request) {
+	log.Printf("%s, %s, %q", req.RemoteAddr, req.Method, req.URL.String())
+
+	postBody, err := json.Marshal(map[string]interface{}{
+		"jsonrpc": "2.0",
+		"method":  "eth_syncing",
```

Comment:
`eth_syncing` returns `false` if the node is synced. If the node is synchronizing to the blockchain, it returns information about how far it has made progress.

To get current block number, use `eth_blockNumber`.

Most importantly, response on `/status` endpoint should have this format:
```json
{
    "synchronizing": false,
    "synchronizationState": {},
    "blockNumber": 13282394
}
```

Reply:
I will add this in next update